io.github.aldrinzigmundv.InnerMirror.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id": "io.github.aldrinzigmundv.InnerMirror",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "innermirror",
+    "finish-args" : [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=ipc"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "env" : {
+            "CARGO_HOME" : "/run/build/rust-flatpak/cargo"
+        }
+    },
+    "modules": [
+        {
+            "name": "innermirror",
+            "buildsystem": "meson",
+            "run-tests": true,
+            "build-commands": [
+                "cargo --offline fetch --manifest-path Cargo.toml --verbose",
+                "cargo --offline build --release --verbose",
+                "install -Dm755 ./target/debug/rust-flatpak -t /app/bin/",
+                "install -Dm644 ./data/${FLATPAK_ID}.metainfo.xml.in.in -t /app/share/metainfo/",
+                "install -Dm644 ./data/${FLATPAK_ID}.desktop.in.in -t /app/share/applications/",
```

Comment:
this is invalid for a desktop file. the `.in.in` is supposed to bo processeded twice by the meson build script.